ui/app/scripts/app/dialogs/module-info.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal-header">
```

Comment:
Style it similarly to:

http://0.0.0.0:8000/#/tasks/modules/timestamp

So we have the striping in place.